drivers/main-device.js

```diff
@@ -282,11 +270,14 @@ module.exports = class mainDevice extends Homey.Device {
                 const blower2_val = blower2.value === 'HIGH';
                 await this.setValue('action_blower_state.2', blower2_val, check);
             }
+            if(heater) {
```

Comment:
This would now break the ability to toggleHeaterMode between REST and Ready, more on PR comment about this.